qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -569,6 +570,20 @@ def _create_cliff_2q_layer_2():
 _NUM_LAYER_1 = 20
 _NUM_LAYER_2 = 16
 
+# Construct a dense multiplication table
```

Comment:
We can replace the generation with the line
```
_clifford_num_to_dense = {1: 0,
 2: 1,
 3: 2,
 4: 3,
 8: 4,
 12: 5,
 16: 6,
 2336: 7,
 2880: 8,
 3520: 9,
 3523: 10,
 5760: 11,
 7680: 12,
 7692: 13}
```
and comment out the generation code. That makes the startup a bit faster.

Reply:
I prefer having the generation code prioritizing the maintenability.